code/game/objects/items.dm

```diff
@@ -73,6 +73,7 @@ GLOBAL_DATUM_INIT(welding_sparks, /mutable_appearance, mutable_appearance('icons
 	/// Percentage armour reduction, happens before flat armour reduction.
 	var/armour_penetration_percentage = 0
 	var/list/allowed = null //suit storage stuff.
+	var/max_suit_w = WEIGHT_CLASS_BULKY
```

Comment:
this needs to be scoped better, we shouldn't add a item level variable if we're only going to use it in a couple very specific cases

Reply:
I added it as a variable to allow for overriding in the future if someone wants to. I basically wholesale copied what I did with belt storage since "suit storage" doesnt technically count as normal storage, so I couldnt access the override.

Reply:
This should probably be at the `/obj/item/clothing/suit` level.

Reply:
auto-doc'd and moved to `clothing.dm`